src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,78 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Finding Installed Applications
```

Comment:
"Finding" can also be done via search through the Launcher, which isn't covered until the next section. "Browsing" might be a more accurate term for what you're describing.